ADR/0006-log-conventions.md

```diff
@@ -137,7 +134,7 @@ want to for the goals of this ADR.
   logging mechanism already conforms to the prescribed format, and can be used, it should be
   _easier_ for teams to conform to this ADR than would be otherwise.
 
-  For example, using the standard controller-runtime logging mechanism will produce a text string in
+  For example, using the standard controller-runtime logging mechanism will produce a json string in
```

Comment:
To avoid (slightly) different logs configuration between operators, it'd be great to have a common configuration of the logger that could be directly copied into each operator.

Reply:
Agree, I just was asking to document default approach for all controllers

Reply:
Thanks, we'll take into account